test/integration/application-ui5-tooling-error-handling/karma-failOnEmptyTestPage-false.conf.js

```diff
@@ -19,7 +19,6 @@ module.exports = function(config) {
 module.exports.shouldFail = false;
 module.exports.assertions = ({expect, log}) => {
 	expect(log).toMatch(/Executed 1 of 1/);
-	expect(log).toMatch(/TOTAL: 1 SUCCESS/);
```

Comment:
The total count is different when multiple browsers are configured (like in the SacueLabs scenario).

Reply:
would it make sense to check if the number is bigger than 0?

Reply:
It's already checked whether the run succeeds. And karma by default fails when there are 0 tests executed.
So I think this check is not necessarily required.